bitrixSM/internal/config/config.go

```diff
@@ -0,0 +1,9 @@
+package config
+
+type Config struct {
+	Port            string `env:"PORT" envDefault:"8080"`
```

Comment:
sometimes it is easier to check env variable before loading and crash application if some of required variables are not defined, instead of troubleshooting why does system does not work or send data but receiver does not receive requests (becouse somebody forget to set up env variable and now it settled automatically to some weird default value :) )